<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QToolButton" name="panelButton">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>450</width>
     <height>550</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>expeyes2.png</normaloff>expeyes2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>446</width>
     <height>544</height>
    </size>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>9</y>
     <width>563</width>
     <height>565</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>370</y>
      <width>401</width>
      <height>38</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="A0Check">
       <property name="font">
        <font>
         <family>Courier 10 Pitch</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>A0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="A1Check">
       <property name="font">
        <font>
         <family>Courier 10 Pitch</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>A1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="lisCheck">
       <property name="font">
        <font>
         <family>Courier 10 Pitch</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Lis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="fitCheck">
       <property name="font">
        <font>
         <family>Courier 10 Pitch</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Fit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="font">
        <font>
         <family>Courier 10 Pitch</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ftButton">
       <property name="font">
        <font>
         <family>Courier 10 Pitch</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>FT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="xmButton">
       <property name="font">
        <font>
         <family>Courier</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>XM</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitButton">
       <property name="font">
        <font>
         <family>Courier</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="helpBrowser">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>409</y>
      <width>405</width>
      <height>136</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Courier 10 Pitch</family>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="plotWidget" name="graphWidget" native="true">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>5</y>
      <width>405</width>
      <height>305</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:wheat;</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>309</y>
      <width>418</width>
      <height>69</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="maximum">
        <number>9</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="msgLabel">
       <property name="font">
        <font>
         <family>Courier 10 Pitch</family>
         <pointsize>7</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Messages         </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLineEdit" name="ID1_display">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>44</y>
     <width>75</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#ddffaa;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ID0_display">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>17</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#ddffaa;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SQR2_display">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>243</y>
     <width>75</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#ddffaa;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CS_display">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>177</y>
     <width>75</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#ddffaa;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="A0_display">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>211</y>
     <width>75</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#ddffaa;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="A1_display">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>243</y>
     <width>75</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#ddffaa;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="A2_display">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>279</y>
     <width>75</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#ddffaa;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SEN_display">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>314</y>
     <width>75</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#ddffaa;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SEN_display_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>348</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#ddffaa;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="experiencesButton">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>520</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Liberation Mono</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#ddffaa;</string>
   </property>
   <property name="text">
    <string>EXPERIENCES</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SQR1_edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>177</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="SQR2_edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>211</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="PULSE_edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>314</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="UPV_edit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>44</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="BPV_edit">
   <property name="geometry">
    <rect>
     <x>281</x>
     <y>79</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="CS_edit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>146</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="FREQ_display">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>486</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#aaaaaa;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="OD0_check">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>79</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#aaaaaa;</string>
   </property>
   <property name="text">
    <string>LO</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="OD1_check">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>115</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#aaaaaa;</string>
   </property>
   <property name="text">
    <string>LO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ID0_button_F">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>19</y>
     <width>15</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>F</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SEN_button_F">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>350</y>
     <width>15</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>F</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AMPLI_button_F">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>488</y>
     <width>15</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>F</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ID0_button_pcent">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>19</y>
     <width>15</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>plotWidget</class>
   <extends>QWidget</extends>
   <header>plotwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
